mails/themes/modern/core/order_conf.html.twig

```diff
@@ -178,18 +178,18 @@
                                       <th bgcolor="#FDFDFD">{{ 'Quantity'|trans({}, 'Emails.Body', locale)|raw }}</th>
                                       <th bgcolor="#FDFDFD">{{ 'Total price'|trans({}, 'Emails.Body', locale)|raw }}</th>
                                     </tr>
-                                    {% if templateType == 'html' %}
-{products}

```

Comment:
I think the indentation is now broken. Can we fix it?

Reply:
Mmmm ... interesting.
Do you think the fact there is blank space or not has an impact on how Mail clients would render the mail?

Reply:
🤔 I don't think so, now we follow the table indentation, in theory there should be no issues on rendering the html generated